DQMOffline/Alignment/python/ALCARECOTkAlDQM_cff.py

```diff
@@ -57,6 +58,40 @@
 #ALCARECOTkAlZMuMuDQM = cms.Sequence( ALCARECOTkAlZMuMuTrackingDQM + ALCARECOTkAlZMuMuTkAlDQM + ALCARECOTkAlZMuMuHLTDQM )
```

Comment:
Are all the commented code lines in this file necessary? I understand they were not added in this PR but maybe we can clean up a bit...

Reply:
Given the time necessary for the re-testing, review, etc., and considering this needs to be backported I would prefer to clean up in a separate PR if there are no comments on the part added here.

Reply:
Fair enough